src/jenkins.py

```diff
@@ -311,19 +311,20 @@ def _install_plugins(
         JenkinsPluginError: if an error occurred installing the plugin.
     """
     command = ["java"]
+    proxy_dict = proxy_config.to_str_dict() if proxy_config else {}
```

Comment:
hmm, why use a dict here? Why not use the data structure directly so that we get autocomplete etc?

Reply:
Changed, thank you!